README.md

```diff
@@ -1,8 +1,53 @@
-# Jarvis Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) In-progress
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+# Nazanin Akbari . Jarvis Consulting
 
+I am a graduate of Concordia University's software engineering program, and I have recently joined Jarvis as a software and data engineer. During Master's program, I spent two years to develop multiple projects with Matlab, Python and C# in different teams of developers. I am fluent in French language. I love programming and computer science because it is a language that can apply to every facet of life. Software provides significant benefits to everything in our lives, and it's critical to have a deep knowledge of it.
```

Comment:
I graduated from Concordia University's Software Engineering program and recently joined Jarvis as a software and data engineer developing projects using A, C, D. During my Master's program, I spent two years developing multiple projects with Matlab, Python, and C# in different developers. I am fluent in French. I love programming and computer science because it is a language that can apply to every facet of life. Soft skill...Agile, team, collb, comm, leader, probelm solving..

Reply:
https://github.com/jarviscanada/jarvis_data_eng_NarimanAlimuradov
https://github.com/jarviscanada/jarvis_data_eng_JasmeetBrar
https://github.com/jarviscanada/jarvis_data_eng_AlexKirilenko
https://github.com/jarviscanada/jarvis_data_eng_FatimaAnwer
https://github.com/jarviscanada/jarvis_data_eng_BrandonDicicco
https://github.com/jarviscanada/jarvis_data_eng_JohnCao
https://github.com/jarviscanada/jarvis_data_eng_TomasRotbauer
https://github.com/jarviscanada/jarvis_data_eng_TinaLee